lib/bandiera/caching_feature_service.rb

```diff
@@ -67,5 +65,10 @@ def update_feature(group, feature_name, feature_data)
 
       super
     end
+
+    private
```

Comment:
No point doing this (and putting them at the bottom) - you can't make constants private.

```
10:47 $ be pry
[1] pry(main)> class Foo
[1] pry(main)*   private
[1] pry(main)*
[1] pry(main)*   BAR=12
[1] pry(main)* end
=> 12
[2] pry(main)> Foo::BAR
=> 12
```

Reply:
Best to leave them at the top of the class, this is usually where most people expect to find them too.

Reply:
> you can't make constants private

I did not know that. It does kill me slightly inside.